components/aws/sagemaker/tests/integration_tests/component_tests/test_workteam_component.py

```diff
@@ -0,0 +1,66 @@
+import pytest
+import os
+import json
+import utils
+from utils import kfp_client_utils
+from utils import sagemaker_utils
+
+def create_workteamjob(
+    kfp_client, experiment_id, region, sagemaker_client, test_file_dir
+):
+
+    download_dir = utils.mkdir(os.path.join(test_file_dir + "/generated"))
+    test_params = utils.load_params(
+        utils.replace_placeholders(
+            os.path.join(test_file_dir, "config.yaml"),
+            os.path.join(download_dir, "config.yaml"),
+        )
+    )
+
+    # Get the account, region specific user_pool and client_id for the Sagemaker Workforce. 
+    test_params["Arguments"]["user_pool"], test_params["Arguments"]["client_id"], test_params["Arguments"]["user_groups"] = sagemaker_utils.get_cognito_member_definitions(sagemaker_client)
+
+    # Generate random prefix for workteam_name to avoid errors if resources with same name exists
+    test_params["Arguments"]["team_name"] = workteam_name = (
+        utils.generate_random_string(5) + "-" + test_params["Arguments"]["team_name"]
+    )
+
+    _ = kfp_client_utils.compile_run_monitor_pipeline(
+        kfp_client,
+        experiment_id,
+        test_params["PipelineDefinition"],
+        test_params["Arguments"],
+        download_dir,
+        test_params["TestName"],
+        test_params["Timeout"],
+    )
+
+    # Delete generated files
+    utils.remove_dir(download_dir)
```

Comment:
can we remove this directory after assertions are successful, it will be helpful in case of failures

Reply:
ack, good point

Reply:
Done.